pkg/channelserver/channelserver.go

```diff
@@ -61,12 +62,21 @@ func (d *DynamicInterval) Wait(ctx context.Context) bool {
 				return true
 			}
 			continue
+		case msg := <-action:
+			if msg == "refresh" {
```

Comment:
Nit: use constant for `refresh` as it is used twice.

Reply:
updated